android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/scenario/LocalWebSocketServerScenario.java

```diff
@@ -66,9 +87,19 @@ public void onStarted() {
         public void onFatalError() {
             mIoHandler.post(mCallbacks::onScenarioError);
         }
+
+        @Override
+        public void onNoConnectionTimeoutReached() {
+            if (mCallbacks instanceof Callbacks)
```

Comment:
NIT: can we always include the braces, even if not syntactically strictly necessary? Even if we avoid a bug once due to this, it's worth the extra characters.